packages/ui/Touchable/tests/Touchable-specs.js

```diff
@@ -245,22 +245,19 @@ describe('Touchable', () => {
 			expect(actual).toEqual(expected);
 		});
 
-		// TODO This test is unstable. `fireEvent` does not recognize the timeStamp property
-		test.skip('should be called before onClick on mouse up', () => {
+		test('should be called before onClick on mouse up', () => {
 			const Component = Touchable({activeProp: 'active'}, DivComponent);
 			const handler = jest.fn();
 			render(<Component onClick={handler} onTap={handler} />);
 			const component = screen.getByTestId('component');
 
-			const ev = {
-				// a matching timeStamp is used by Touchable to prevent multiple onTaps on "true"
-				// click (mouseup + click)
-				timeStamp: 1
-			};
-			const clickEv = {};
-			fireEvent.mouseDown(component, ev);
-			fireEvent.mouseUp(component, ev);
-			fireEvent.click(component, clickEv);
+			const mouseDownEvent = createEvent.mouseDown(component, {ev:{timeStamp: 1}});
+			const mouseUpEvent = createEvent.mouseUp(component, {ev:{timeStamp: 1}});
+			const clickEvent = createEvent.click(component, {ev:{timeStamp: 1}});
```

Comment:
When I test, the test sometimes fail, I think it's because timeStamps were not set to 1 on those events.
I think we need to set timeStamp using Object.defineProperty.
And it might be helpful if we keep a comment related to timestamp.

```suggestion
			const mouseDownEvent = createEvent.mouseDown(component, {});
			const mouseUpEvent = createEvent.mouseUp(component, {});
			const clickEvent = createEvent.click(component, {});

			// a matching timeStamp is used by Touchable to prevent multiple onTaps on "true"
			// click (mouseup + click)
			Object.defineProperty(mouseDownEvent, 'timeStamp', {value: 1});
			Object.defineProperty(mouseUpEvent, 'timeStamp', {value: 1});
			Object.defineProperty(clickEvent, 'timeStamp', {value: 1});
```

Reply:
Done